src/rgb_packet_processor.cpp

```diff
@@ -50,21 +52,31 @@ void RgbPacketProcessor::setFrameListener(libfreenect2::FrameListener *listener)
 }
 
 DumpRgbPacketProcessor::DumpRgbPacketProcessor()
+  : frame(new Frame(1920, 1080, 4))
 {
 }
 
 DumpRgbPacketProcessor::~DumpRgbPacketProcessor()
 {
+  delete frame;
 }
 
 void DumpRgbPacketProcessor::process(const RgbPacket &packet)
 {
-  //std::stringstream name;
-  //name << packet->sequence << "_" << packet->unknown0 << "_" << jpeg_buffer_length << ".jpeg";
-  //
-  //std::ofstream file(name.str().c_str());
-  //file.write(reinterpret_cast<char *>(packet->jpeg_buffer), jpeg_buffer_length);
-  //file.close();
+  frame->sequence = packet.sequence;
+  frame->timestamp = packet.timestamp;
+  frame->width = 1;
+  frame->height = 1;
+  frame->bytes_per_pixel = packet.jpeg_buffer_length;
+  frame->data = packet.jpeg_buffer;
```

Comment:
Need memcpy here too. Same reason as in `DumpDepthPacketProcessor::process()`. But I think here it needs allocating with a fixed size `new Frame(1, 1, 1920*1080*4);` (packet.jpeg_buffer_length varies, and using it creates memory fragmentation).